timezone/src/modules/Timezone.rb

```diff
@@ -984,6 +986,23 @@ def Summary
       HTML.List(ret)
     end
 
+
+    # Checks whether the system has Windows installed on a primary partition with NTFS
+    def system_has_windows?
+      storage = Y2Storage::StorageManager.instance
+      Storage::Ntfs::all(storage.probed).each do |ntfs|
+        ntfs.blk_devices.each do |blk_device|
+          if Storage::partition?(blk_device)
+            if Storage::to_partition(blk_device).type == Storage::PartitionType_PRIMARY
+              return true if ntfs.detect_content_info.windows?
+            end
+          end
+        end
```

Comment:
I mean, in a random YaST module, checking if a devicegraph contains a windows installation (a kind of simple question) should not imply four occurrences of ``Storage::`.